gc/verbose/VerboseWriterFileLoggingBuffered.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2020 IBM Corp. and others
```

Comment:
a few flies does not seem to be changed anymore after the last updates, so no need to change copyright year either